packages/vm/tests/BlockchainTestsRunner.ts

```diff
@@ -5,6 +5,7 @@ import { Block } from '@ethereumjs/block'
 import Blockchain from '@ethereumjs/blockchain'
 import { setupPreConditions, verifyPostConditions } from './util'
 import Common from '@ethereumjs/common'
+import { TransactionFactory } from '../../tx/dist'
```

Comment:
small nit, I'm sure VS Code automatically did this import hehe
```suggestion
import { TransactionFactory } from '@ethereumjs/tx'
```